docs/detections/rules-ui-manage.asciidoc

```diff
@@ -135,9 +135,10 @@ Similarly, rules will be skipped if they can't be modified by a bulk edit. For e
 * Bulk edit multiple rules: Select the rules you want to edit, then select an action from the *Bulk actions* menu:
 ** *Index patterns*: Add or delete the index patterns used by all selected rules.
 ** *Tags*: Add or delete tags on all selected rules.
-** *Add rule actions*: Add <<rule-notifications,notification actions>> on all selected rules.
+** *Add rule actions*: Add <<rule-notifications,rule actions>> on all selected rules. If you add multiple actions, you can specify action frequency for each action. If you don't want to change the frequency of existing actions on certain rules, update those rules separately.
```

Comment:
I'm not sure I understand the last sentence `If you don't want to change the frequency of existing actions on certain rules, update those rules separately.`. This operation doesn't change frequency of existing actions. This operation will only add new actions to selected rules.

Reply:
@e40pud it could be referring to the option to overwrite existing actions on a rule (see screenshot below). I can make the sentence more direct. 

<img width="600" alt="Screenshot 2023-05-17 at 10 36 59 AM" src="https://github.com/elastic/security-docs/assets/79124755/75395b02-aee6-480a-b133-204e9aa2335f">


For example:

```suggestion
** *Add rule actions*: Add <<rule-notifications,rule actions>> on all selected rules. If you add multiple actions, you can specify action frequency for each action. To overwrite the frequency of existing actions select the option to **Overwrite all selected rules actions**.
```